docs/.vuepress/handsontable-manager/index.js

```diff
@@ -0,0 +1,19 @@
+const { useHandsontable } = require('./use-handsontable');
+const { getDependencies } = require('./dependencies');
+const { register } = require('./register');
+
+if (module) {
+  module.exports = {
+    useHandsontable,
+    instanceRegister: register,
+    getDependencies,
+    applyToWindow: () => {
+      /* eslint-disable no-restricted-globals */
+      if ((typeof window !== 'undefined')) {
+        window.useHandsontable = useHandsontable;
+        window.instanceRegister = register;
+      }
```

Comment:
We have Vue application at our disposal, and have examples container implemented as component. Shouldn't helpers be imported instead being global? Not the best practice in Vue app I guess

Reply:
cc @budnix

Reply:
It doesn't hold back the initial release. Let's do it at your earliest convenience @Siemienik

Reply:
REported https://github.com/handsontable/handsontable/issues/8430.